tools/sof-dump-status.py

```diff
@@ -82,6 +86,16 @@ def loadDMI(self):
             self.dmi[name] = line[idx:]
         pass
 
+    def loadFw(self):
+        self.fw_lst.clear()
+        exit_code, output = subprocess.getstatusoutput("cat /sys/devices/soc0/soc_id")
+        if exit_code != 0:
+            return
```

Comment:
I can't really use stdout for debug/error messages because of the way the script is used but I could try using stderr (altough I don't think it's the best solution for debug messages). Another option would be to use some sort of log file.